test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -848,6 +848,16 @@ def forward(self, input):
         model = CumSum()
         self.run_test(model, x)
 
+    def test_baddbmm(self):
+        class MyModule(torch.nn.Module):
+            def forward(self, input, batch1, batch2):
+                return torch.baddbmm(input, batch1, batch2, alpha=torch.tensor(5), beta=3.5)
```

Comment:
does it export correctly with alpha and beta as inputs of the model?
if so could we add a test that captures it?

Reply:
Yes, @lara-hdr 's comments make sense.

Reply:
Yes, I added the test